openhub_django/variables.py

```diff
@@ -0,0 +1,2 @@
+OH_TOKEN = ''
+ORG_NAME = ''
```

Comment:
@jayvdb, where should we put these environment variables, so that it would configurable for the orgs who install this package?

Reply:
Since, they're being treated as environment variables we can make use of `python os` module to retrieve there values from operating system environment easily. The same approach is being used by coala webservices and community repositories.